eregs.py

```diff
@@ -21,7 +23,7 @@ def cli(debug):
     if debug:
         log_level = logging.DEBUG
         sys.excepthook = lambda t, v, tb: ipdb.post_mortem(tb)
-    coloredlogs.install(level=log_level, fmt="%(levelname)s %(message)s")
+    coloredlogs.install(level=log_level)
```

Comment:
IMHO, the default format was more informative as it gives visibility into the module where the message was emitted. Also, any desired format can be set using the envvar COLOREDLOGS_LOG_FORMAT. My hope is that we'll eventually provide a default log config or set the COLOREDLOGS_LOG_FORMAT envvar explicitly.